test-core/build.gradle

```diff
@@ -1,16 +1,17 @@
+plugins {
+    id "io.micronaut.build.internal.module"
+}
+
 dependencies {
-    annotationProcessor "io.micronaut:micronaut-inject-java:$micronautVersion"
-    compileOnly "io.micronaut:micronaut-http-server:$micronautVersion"
-    compileOnly("org.spockframework:spock-core:$spockVersion") {
-        exclude group: "org.codehaus.groovy", module: "groovy-all"
-    }
+    annotationProcessor "io.micronaut:micronaut-inject-java"
```

Comment:
I don't think this is necessary.

Reply:
Indeed it isn't: https://github.com/micronaut-projects/micronaut-build/blob/278c63890a5457637a66eaef8d6dfe59addcf291/src/main/groovy/io/micronaut/build/MicronautModulePlugin.groovy#L22-L32

Reply:
Thanks both, fix pushed: https://github.com/micronaut-projects/micronaut-test/pull/518/commits/22d96a2e9d1fa1e90c575a354e60ecf24ac74701